app/perfunwind.cpp

```diff
@@ -440,10 +441,14 @@ void PerfUnwind::resolveCallchain()
     bool addedUserFrames = false;
     PerfSymbolTable *symbols = symbolTable(m_currentUnwind.sample->pid());
 
-    auto reportIp = [&](quint64 ip) -> bool {
+    auto reportIp = [&](quint64 ip, bool branchStack) -> bool {
```

Comment:
what are the changes here doing? please explain this in a separate commit - I don't follow